src/file/mod.rs

```diff
@@ -543,6 +543,10 @@ impl NamedTempFile {
     /// platforms so please read the security section of this type's
     /// documentation.
     ///
+    /// On Unix systems, the file is created with `600` permissions, instead of the default `644`,
```

Comment:
- Let's put this in a new "permissions" section.
- The default isn't necessarily 0644, it depends on the user's mode.
- The leading 0 is important (it means that the number is on octal).